airflow/bin/cli.py

```diff
@@ -84,14 +84,15 @@ def trigger_dag(args):
     session = settings.Session()
     # TODO: verify dag_id
     execution_date = datetime.now()
+    run_id = args.run_id or "manual__{0}".format(datetime.datetime.utcnow().isoformat())
```

Comment:
btw just saw that we do import `datetime`.. https://github.com/airbnb/airflow/blob/master/airflow/bin/cli.py#L6

anyways will update the PR to use `execution` date..